docs/rules/padding-around-before-all-blocks.md

```diff
@@ -0,0 +1,29 @@
+# padding-around-before-all-blocks
+
+## Rule Details
+
+This rule enforces a line of padding before _and_ after `beforeAll` statements
+
+Note that it does _not_ enforce a line of padding between a `beforeAll` and a closing bracket in a block statement
```

Comment:
Another way to say this might be: "it doesn't add/enforce a padding line if it's the last statement in its scope". I think that's a _tiny_ bit more precise, because it also covers if `beforeAll` is the last item (for some reason) in the program (file). Total nitpick. Your call!